### ISSUE TITLE
Compilation fails on aarch64

### ISSUE THREAD

**Issue by parkerlreed** [yvt/openspades#906]
Possibly related to #416. New issue because different architecture and the only solution back then was a fork which is now outdated

Log https://gist.github.com/parkerlreed/5ee2e088851036eb8f500bfb7c8a3011

**Comment by parkerlreed** [comment]
As suggested on that issue, I tried adding stk's Angelscript arm snippet to the current openspades but it still bails trying the x64 codepath.

https://github.com/supertuxkart/stk-code/blob/master/lib/angelscript/projects/cmake/CMakeLists.txt#L115

**Comment by Haxk20** [comment]
Yes this is because OpenSpades uses a VERY old version of AngelScript. 2.31.0 which is from 2016 and the latest is 2.33 from 2018 which includes many fixes for the aarch64.

I will fork OS today and see if i can update the AngelScript version to the latest version.

**Comment by Haxk20** [comment]
BTW this is how supertuxkart got it fixed https://github.com/supertuxkart/stk-code/issues/2569

**Comment by CloudyProton** [comment]
The same issue occurs when building on ppc64el 

```
/usr/bin/ld: ../lib/libAngelscript.a(as_callfunc.cpp.o): in function CallSystemFunction(int, asCContext*)':
/home/User/OpenSpades/openspades/Sources/AngelScript/source/as_callfunc.cpp:712: undefined reference to CallSystemFunctionNative(asCContext*, asCScriptFunction*, void*, unsigned int*, void*, unsigned long&, void*)'
collect2: error: ld returned 1 exit status
make[2]: *** [Sources/CMakeFiles/OpenSpades.dir/build.make:3562: bin/openspades] Error 1
make[1]: *** [CMakeFiles/Makefile2:230: Sources/CMakeFiles/OpenSpades.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

**Comment by yvt** [comment]
Removing the bug label as it's not a bug but just a lack of support.

**Comment by tigercoding56** [comment]
any activity on solving this  issue ?
